examples/powertools-examples-core/terraform/main.tf

```diff
@@ -0,0 +1,4 @@
+# terraform modules
```

Comment:
How do we know that we haven't broken this - can we lint it as part of the build?

Reply:
could we do a `terraform validate` at least ?

Reply:
@scottgerring we should something similar for SAM and the others... `sam build` / `sam validate --lint --template template.yml`

Reply:
ok I will add some form of linting/validation